crates/bevy_ecs/src/world/entity_ref.rs

```diff
@@ -255,11 +255,12 @@ impl<'w> EntityMut<'w> {
 
     // TODO: move to BundleInfo
     pub fn remove_bundle<T: Bundle>(&mut self) -> Option<T> {
+        let mut to_remove = Vec::new();
```

Comment:
Would like some opinions on this, I'm sure I could've made it work by operating on the components/storages directly, but I was not confident and this seems simpler anyways.